proposals/stage-1/SUPPORTER_PROGRAM/SUPPORTER_PROGRAM.md

```diff
@@ -0,0 +1,51 @@
+## The People's Community of JavaScriptlandia 
+
+JavaScriptLandia is a place where all JavaScript developers, whether if you're all about TypeScript, a React evangelist, or you stick to Node.js gather to declare their support for the JavaScript ecosystem and OpenJS. 
+
+It's an Individual Membership Program whose goals are to help bring awareness about OpenJS projects to possible contributors. A way for individuals to show support and be a part of the OpenJS foundation, as well as providing an opportunity for community with the other members. 
```

Comment:
> ... be a part of the OpenJS foundation

Foundation should be capitalized when referring to the organization. This is also repeated a few times through this doc.